types/relay-runtime/lib/util/NormalizationNode.d.ts

```diff
@@ -55,6 +41,14 @@ export interface NormalizationScalarField {
     readonly storageKey: string | null | undefined;
 }
 
+export type NormalizationFlightField = {
+    readonly kind: string; // 'FlightField';
```

Comment:
why `string` over `FlightField` here?

Reply:
these "kind" fields break things.. The general gist here is to keep them as string.